coral/src/app/features/topics/list/TopicCard.tsx

```diff
@@ -0,0 +1,57 @@
+import { BorderBox, Grid, GridItem } from "@aivenio/design-system";
+
+const exampleCard = {
+  topicid: 1010,
+  sequence: "308",
+  totalNoPages: "1",
+  currentPage: "1",
+  allPageNos: ["1"],
+  topicName: "aivtopic1",
+  noOfPartitions: 1,
+  description: "Topic description",
+  documentation: null,
+  noOfReplcias: "2",
+  teamname: "Ospo",
+  cluster: "1",
+  clusterId: null,
+  environmentsList: ["DEV", "TST"],
+  showEditTopic: false,
+  showDeleteTopic: false,
+  topicDeletable: false,
+};
+
+function TopicCard() {
```

Comment:
Yep! We can't get the same content in there, since it's doesn't take children. 
Also it's a `div` structure and I wanted to use a `list`, because it seems like more people favor a list structure for accessibility reasons. Assistive technology then convey the information that this is a list of things with x items for example, which is nice. 

An alternative can be to use an `article` tag for the cards, which I feel makes more sense if there's more content in it. Also it does not give the additional information to the user how may items there are, but that could be text content on the page (and at some point probably will).